patches/0002-Add-crypto-backend-foundation.patch

```diff
@@ -702,6 +956,118 @@ index be8ac30f9daf9f..3356154c7d1082 100644
  	< crypto/rand
  	< crypto/ed25519
  	< encoding/asn1
+diff --git a/src/net/http/client_test.go b/src/net/http/client_test.go
+index 0fe555af38f08a..5b2c8051a8ba83 100644
+--- a/src/net/http/client_test.go
++++ b/src/net/http/client_test.go
+@@ -960,7 +960,7 @@ func testResponseSetsTLSConnectionState(t *testing.T, mode testMode) {
+ 
+ 	c := ts.Client()
+ 	tr := c.Transport.(*Transport)
+-	tr.TLSClientConfig.CipherSuites = []uint16{tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA}
++	tr.TLSClientConfig.CipherSuites = []uint16{tls.TLS_RSA_WITH_AES_256_GCM_SHA384}
```

Comment:
To make sure I understand the change: it isn't all that important to this test which cipher is chosen, so this change is simply making it use a cipher compatible with FIPS TLS because there's no reason not to, and it's simpler than Abandon/Force? (Especially since these changes are making their way upstream.)

Reply:
Yep, that's it.